CMakeLists.txt

```diff
@@ -46,6 +46,15 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
     set(J2V8_LIB_PLATFORM_NAME "linux")
     set(J2V8_LIB_PREFIX "")
     set(J2V8_LIB_ARCH_NAME "x86")
+
+    # TODO: This hack could be used to have a different lib filename for alpine, e.g. libj2v8_alpine_x86_64
```

Comment:
I just realized that this hack IS required. Otherwise there is no possibility to deploy, since building the alpine version would overwrite the regular linux version!